client/components/focusable/style.scss

```diff
@@ -1,5 +1,3 @@
-.focusable {
-	.accessible-focus &:focus {
-		box-shadow: 0 0 0 2px var( --color-primary-light );
-	}
+.accessible-focus .focusable:focus {
```

Comment:
this seems like the right transform (yay much more readable), but where does `accessible-focus` come from? I don't see it in the component or mentioned in the README.

Reply:
ah, `lib/accessible-focus` is providing it on the `html` element